src/storage/src/hummock/store/version.rs

```diff
@@ -287,24 +332,139 @@ impl HummockReadVersion {
                 let max_committed_epoch = committed_version.max_committed_epoch();
```

Comment:
> I think we can push `VersionUpdate::CommittedDelta` on receiving a version delta and implement L328, which can simplify the logic of finding and replacing the relevant objects without traversing the all the sub-levels.

No, we can't. Because we may receive a version instead of a version delta from meta.